prime-router/docs/getting-started/install-openjdk.md

```diff
@@ -15,6 +15,37 @@ VERSION=11
 brew install openjdk@${VERSION?}
 ```
 
+If after running `./cleanslate.sh` you see the following error
```

Comment:
Make a mention that this is on Mac OS.  There are people working on Linux and Windows as well.

Reply:
Hey I missed this in all my other emails but this is under the Mac OS section of instructions so I think it should be clear to the Linux and Windows users.